server/resp/src/main/java/org/infinispan/server/resp/logging/Log.java

```diff
@@ -0,0 +1,36 @@
+package org.infinispan.server.resp.logging;
+
+import static org.jboss.logging.Logger.Level.WARN;
+
+import java.nio.CharBuffer;
+
+import org.infinispan.commons.CacheConfigurationException;
+import org.jboss.logging.BasicLogger;
+import org.jboss.logging.annotations.Cause;
+import org.jboss.logging.annotations.LogMessage;
+import org.jboss.logging.annotations.Message;
+import org.jboss.logging.annotations.MessageLogger;
+
+/**
+ * Log abstraction for the Resp protocol server module. For this module, message ids
+ * ranging from 12001 to 13000 inclusively have been reserved.
+ *
+ * @author William Burns
+ * @since 14.0
+ */
+@MessageLogger(projectCode = "ISPN")
+public interface Log extends BasicLogger {
+   @Message(value = "Cache '%s' has expiration enabled which violates the Resp protocol", id = 12001)
+   CacheConfigurationException invalidExpiration(String cacheName);
+
+   @Message(value = "Cannot enable authentication without an authenticator", id = 12002)
+   CacheConfigurationException authenticationWithoutAuthenticator();
+
+   @LogMessage(level = WARN)
+   @Message(value = "Unexpected exception encountered", id = 12003)
```

Comment:
```suggestion
   @Message(value = "Encountered an unexpected exception.", id = 12003)
```

Reply:
Are exceptions ever expected? What is the consequence of this? Does an operation fail and the user has to retry?

Would it be sensible to expand a little and say "Encountered an exception and the operation might not have completed successfully."

Reply:
We handle exceptions in the code directly for the cases we know of. This log message is for ones that are out of the scope of the handler. Something like an OOM or other things we don't ever want to see.

Reply:
Thanks for explaining. What about something like?

"Received an unexpected exception from the JVM."